designs/virtualization/4.x/cd roms/cd roms.md

```diff
@@ -0,0 +1,39 @@
+---
+parent: Virtualization
+version: 4.x
+---
+# CD ROMs
+
+This PR deals with removing CD-ROMs from the details view and including them in the disks tab.
+
+## CD-ROMs on VM Details page
+
+We can remove CD ROMs from the VM Details view
+
+![CDROMs are on the details page](img/Details.png)
+
+and add a ‘Drive’ column to the ‘Disks’ table view.
+The default type will be ‘Disk’ and the other option(s) will be ‘CD ROM’ and possibly ‘LUN’.
```

Comment:
```suggestion
The default type will be ‘Disk’ and the other option(s) will be ‘CD ROM’.
```
I'd pull LUN out for now since we don't have this yet. We'll likely have to provide some designs around it when it gets prioritized.